src/components/Questions/Question.vue

```diff
@@ -31,8 +31,29 @@
 		<div v-if="!readOnly"
 			class="question__drag-handle"
 			:class="{'question__drag-handle--shiftup': shiftDragHandle}"
-			:aria-label="t('forms', 'Drag to reorder the questions')">
+			:aria-label="t('forms', 'Drag to reorder the questions')"
+			role="button">
+			<NcButton ref="buttonU"
```

Comment:
```suggestion
			<NcButton ref="buttonUp"
```
shouldn't this be buttonUp?